Fix integration test workflow (#508)
kyma-project/api-gateway: docs/contributor/04-30-ci-cd.md

# CI/CD

## Environments

> Some of the jobs that are running in this repository are configured to run in context of a repository environment. This allows ensuring that jobs that need access to repository secrets are manually allowed by repository maintainers.

Reviewer: ```suggestion
In this repository, certain jobs are set up to run within the context of the repository environment. This ensures that the jobs that require access to repository Secrets are manually allowed by repository maintainers.
```

There are two environments configured:

> - 'internal' - used when the job is run by repository collaborator

Reviewer: ```suggestion
- 'internal' - used when a repository collaborator runs a job in the repository
```

> - 'restricted' - used when the job is run by outside collaborator. The run need to by approved by @kyma-project/goat.

Reviewer: ```suggestion
- 'restricted' - used when an outside collaborator runs a job in the repository. The run must be approved by @kyma-project/goat.
```
